src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -145,14 +141,17 @@ export class ExposureNotificationService {
   }
 
   async updateExposureStatusInBackground() {
-    const lastStatus = this.exposureStatus.get();
+    await this.init();
     await this.updateExposureStatus();
     const currentStatus = this.exposureStatus.get();
-    if (lastStatus.type === 'monitoring' && currentStatus.type === 'exposed') {
+    if (currentStatus.type === 'exposed' && !currentStatus.notificationSent) {
```

Comment:
Change logic of showing notification. This making sure that we show exposed notification once.